docs/src/pages.js

```diff
@@ -0,0 +1,301 @@
+import findPages from /* preval */ 'docs/src/modules/utils/findPages';
+
+const pages = [
+  {
+    pathname: '/getting-started',
+    children: [
+      {
+        pathname: '/getting-started/installation',
+      },
+      {
+        pathname: '/getting-started/usage',
+      },
+      {
+        pathname: '/getting-started/example-projects',
+      },
+      {
+        pathname: '/getting-started/page-layout-examples',
+      },
+      {
+        pathname: '/getting-started/learn',
+      },
+      {
+        pathname: '/getting-started/faq',
+        title: 'Frequently Asked Questions',
```

Comment:
These can be removed now.

Reply:
No

Reply:
The strategy is the following;

Reply:
I don't want to have to write all the pages name by hand. The logic is still relying on the smart title generation logic. Nothing changed on this end for the English language.
I have created a script that hydrate the `translations.json` values, ith the english referrence for the translators.

Reply:
Okay, so `title` is still used to regenerate `translations.json` after pages are added or moved, and `translations.json` is used to display the English titles?

One downside is that moves (change of key name) will invalidate the translations of the title of the moved page, unless the key name in the translations is manually updated to match.

Reply:
I thought Crowdin was losing the translations of the page when moving it?

Reply:
I'm not sure, but I believe it should be possible to overcome that if so. It would definitely lose the attribution and approval status.

For the titles, worst case the previous translation would be offered by translation memory, so it's not the end of the world. It's not like it happens often either.